CMakeLists.txt

```diff
@@ -50,6 +50,8 @@ option(ENABLE_WARNINGS "Enable warnings" ON)
 option(ENABLE_NATIVE "Enable native CPU build tuning" OFF)
 option(ENABLE_AVX "Enable AVX support" OFF)
 option(ENABLE_OPENMP "Enable OpenMP" ON)
+option(ENABLE_THREADS "Enable threads" ON)
+option(ENABLE_KOKKOS "Enable Kokkos" ON) # ON by default during development.
```

Comment:
Once we decide to release, this should be off by default unless explicitly stated.

Reply:
For simplicity, in the most recent changes, I removed the ENABLE_THREADS option.
The ENABLE_KOKKOS is probably better to be OFF by default.
I intend to do it before merging. 
It is ON now just for testing, as you know.